solr/solrj/src/java/org/apache/solr/common/params/HighlightParams.java

```diff
@@ -47,6 +47,8 @@
 
   // sizing
   public static final String FRAGSIZE    = HIGHLIGHT+".fragsize"; // OH, FVH, UH
+  public static final String FRAGALIGNRATIO = HIGHLIGHT+".fragAlignRatio"; // UH
+  public static final String FRAGSIZEISMINIMUM = HIGHLIGHT+".fragsizeIsMinimum"; // UH
```

Comment:
very minor: I'd prefer these two added rows are switched so that fragsizeIsMinimum directly follows fragsize